scripts/setup-socorro.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# create DB if it does not exist
+# TODO handle DB not on localhost - could use setupdb for this
+su - postgres -c "psql breakpad -c ''" > /var/log/socorro/setupdb.log 2>&1
+if [ $? != 0 ]; then
+    echo "Creating new DB, may take a few minutes"
+    pushd /data/socorro/application > /dev/null
+    su postgres -c "PYTHONPATH=. /data/socorro/socorro-virtualenv/bin/python \
+        ./socorro/external/postgresql/setupdb_app.py \
+        --alembic_config=/etc/socorro/alembic.ini --database_name=breakpad \
+        --database_superusername=postgres" &> /var/log/socorro/setupdb.log
+    if [ $? != 0 ]; then
+        echo "WARN could not create database on localhost"
+        echo "See /var/log/socorro/setupdb.log for more info"
+    fi
+    popd > /dev/null
+else
+    echo "Running database migrations with alembic"
+    pushd /data/socorro/application > /dev/null
+    su postgres -c "PYTHONPATH=. ../socorro-virtualenv/bin/python \
+        ../socorro-virtualenv/bin/alembic \
+        -c /etc/socorro/alembic.ini upgrade head" &> /var/log/socorro/alembic.log
+    if [ $? != 0 ]; then
+        echo "WARN could not run alembic migrations"
+        echo "See /var/log/socorro/alembic.log for more info"
+    fi
+    popd > /dev/null
+fi
+
+/data/socorro/socorro-virtualenv/bin/python \
+    /data/socorro/webapp-django/manage.py syncdb --noinput \
+    &> /var/log/socorro/django-syncdb.log
+if [ $? != 0 ]; then
+    echo "WARN could not run django syncdb"
+    echo "See /var/log/socorro/django-syncdb.log for more info"
+fi
+
+/data/socorro/socorro-virtualenv/bin/python \
+    /data/socorro/webapp-django/manage.py migrate --noinput \
+    &> /var/log/socorro/django-migrate.log
+if [ $? != 0 ]; then
+    echo "WARN could not run django migration"
+    echo "See /var/log/socorro/django-migrate.log for more info"
+fi
+
+# create ElasticSearch indexes
+echo "Creating ElasticSearch indexes"
```

Comment:
>Concerning scripts/install.sh and scripts/package.sh (and yes, I've said this before), but at some point we'll want to replace FPM with proper packaging. We don't need to fix this today, but we should probably create a bug for it - could be good for an intern or junior?

sgtm, but some things I'd note:

* we're going to start breaking up the repo into separate modules and therefore will need separate packages (planning to do ```socorro-collector``` first)
 * the build system in the current ```socorro``` module is difficult to deal with, it'll be easier with the new split-up modules
* we're probably going to want debian packages too (which was part of the impetus for fpm)
* the new split-up socorro modules are going to be proper python packages, so we might be able to get away from having to do distro-specific packaging (say we used something like https://github.com/progrium/buildstep and had generic treatment of "web" and "worker" classes of app)